src/Main.elm

```diff
@@ -167,8 +178,11 @@ update msg ({ attempts, currentTry } as model) =
         UpdateTry string ->
             ( { model | currentTry = string }, Cmd.none )
 
+        SwitchLanguage lang ->
+            update NewGame { model | language = lang }
```

Comment:
It's usually better to avoid calling `update` recursively, so here we could just return `( { model | language = lang }, Cmd.none )` :)

Reply:
Ah no my bad, we need to `init` indeed. Scratch it, we won't bother destructuring just to update language here 👍